<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_form</class>
 <widget class="QMainWindow" name="main_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>0</height>
          </size>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>124</width>
            <height>161</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="fonButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Фоны</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="regButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Регистрация</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="musicButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Музыка</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="onePlayerButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Одиночная игра</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="commandButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Командная игра</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>142</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>42</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>ВЫ ОРЁТЕ ВЕЛИКОЛЕПНО!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="score_label">
            <property name="font">
             <font>
              <pointsize>42</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(239, 41, 41);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="command_1_label">
                  <property name="text">
                   <string>Команда1</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableWidget" name="cmd_1_tableWidget"/>
                </item>
                <item>
                 <widget class="QWidget" name="widget_6" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QPushButton" name="cmd_1_rstButton">
                     <property name="text">
                      <string>--</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="cmd_1_itogi_text_label">
                     <property name="text">
                      <string>Итог</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="cmd_1_itogi_label">
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="widget_7" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="command_2_label">
                  <property name="text">
                   <string>Команда2</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableWidget" name="cmd_2_tableWidget"/>
                </item>
                <item>
                 <widget class="QWidget" name="widget_8" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QPushButton" name="cmd_2_rst_Button">
                     <property name="text">
                      <string>--</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="cmd_2_itogi_text_label">
                     <property name="text">
                      <string>Итог</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="cmd_2_itogi_label">
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="game_mode_label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="current_player_label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_11" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QPushButton" name="run_game_Button">
                  <property name="text">
                   <string>Начать</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="end_game_Button">
                  <property name="text">
                   <string>Закончить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="exitButton">
            <property name="text">
             <string>Выход</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>24</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_Reg">
   <property name="text">
    <string>Регистрация</string>
   </property>
   <property name="toolTip">
    <string>Регистрация новых участников</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_Music">
   <property name="text">
    <string>Музыка</string>
   </property>
   <property name="toolTip">
    <string>Музыка</string>
   </property>
  </action>
  <action name="action_One_Player">
   <property name="text">
    <string>Одиночная игра</string>
   </property>
   <property name="toolTip">
    <string>Одиночная игра</string>
   </property>
  </action>
  <action name="action_Command_Game">
   <property name="text">
    <string>Командная игра</string>
   </property>
   <property name="toolTip">
    <string>Командная игра</string>
   </property>
  </action>
  <action name="action_Fon">
   <property name="text">
    <string>Фоны</string>
   </property>
   <property name="toolTip">
    <string>Фоны</string>
   </property>
  </action>
  <action name="action_Run_Game">
   <property name="text">
    <string>Начать игру</string>
   </property>
   <property name="toolTip">
    <string>Начать игру</string>
   </property>
  </action>
  <action name="action_End_Game">
   <property name="text">
    <string>Закончить игру</string>
   </property>
   <property name="toolTip">
    <string>Закончить игру</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>action_Exit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onePlayerButton</sender>
   <signal>clicked()</signal>
   <receiver>action_One_Player</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fonButton</sender>
   <signal>clicked()</signal>
   <receiver>action_Fon</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regButton</sender>
   <signal>clicked()</signal>
   <receiver>action_Reg</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>musicButton</sender>
   <signal>clicked()</signal>
   <receiver>action_Music</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandButton</sender>
   <signal>clicked()</signal>
   <receiver>action_Command_Game</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>run_game_Button</sender>
   <signal>clicked()</signal>
   <receiver>action_Run_Game</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>end_game_Button</sender>
   <signal>clicked()</signal>
   <receiver>action_End_Game</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
